rise-financial-es6.js

```diff
@@ -308,14 +311,32 @@
 
       this._goPending = false;
 
-      this._getData(
-        {
-          type: this.type,
-          duration: this.duration,
-        },
-        this._instruments,
-        this.instrumentFields
-      );
+      if ( this._initialGo ) {
+        this._initialGo = false;
+
+        // configure and execute initial request
+        this._getData(
+          {
+            type: this.type,
+            duration: this.duration,
+          },
+          this._instruments,
+          this.instrumentFields
+        );
+      } else {
+        // get cached data (if available)
+        this.$.data.getItem( this._getDataCacheKey(), ( cachedData ) => {
```

Comment:
@stulees If a refresh is pending it doesn't make sense to bother getting the cached data since you're just going to refresh it anyway. I think it would be better to handle that bit separately.